### DISCUSSION TITLE
Tempature in Proxmox GUI

### DISCUSSION THREAD

**Discussion by awptechnologies** [community-scripts/ProxmoxVE#651] [Ideas]
I have found a way to get the cpu temps in the gui. But i am not capable of turning the process into a script and enable automatically setting it up after any kind of pve-manager update or something that triggers the default config to be put back into place.

Below is a step by step of how i enable it for my dellr730xd with 2 processors.
![Screenshot From 2024-12-03 04-26-20](https://github.com/user-attachments/assets/94695715-dc0d-47e4-b9b1-64a165442402)

![Screenshot From 2024-12-03 04-28-51](https://github.com/user-attachments/assets/73498679-69a9-41ee-bc8e-ff788c344157)

**Comment by Banjer** [comment]
There is a script for that, at least tested for 7.x - 8.2
https://github.com/Meliox/PVE-mods
Haven't tried it myself, as i have no need for it. and I'm not a big fan of modifying "system" files.

**Comment by awptechnologies** [comment]
is this worth putting in the community scripts?
